pages/slack.md

```diff
@@ -0,0 +1,18 @@
+title: Slack
+---
+
+# Rules for our Slack channel
+As with any of our conference spaces (online or physical), participants are expected to follow our [Code of Conduct](/code-of-conduct/).
+
+In order to make the experience pleasant for everyone, we also ask that you follow some Slack-specific rules:
+
+1. Do not mention people who don't belong to a particular room in that room.
+2. Please keep job postings to the #jobs channel.
```

Comment:
How about "Try to keep on-topic in each channel, and only post job postings to the #jobs channel."?  I think this should actually be rule 1.